### PR TITLE
Updating changeling for micro release 0.7.4

### PR DESCRIPTION
**Pull request by bam241**
Just updating the changeling for the release of 0.7.4

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
 **New Capabilities**
 
+**Change**
+     
+**Fix**
+
+**Maintenance**
+
+
+v0.7.3
```
Comment (gonuke) [inline CHANGELOG.rst]: Won't this be 0.7.4?
```suggestion
v0.7.4
```

**Review by gonuke — CHANGES_REQUESTED** [review]
One tiny typo (?)...

File: CHANGELOG.rst (hunk 1 above)
Comment (bam241) [reply CHANGELOG.rst]: fixed it before seeing your review.

**Comment by bam241** [comment]
this should be good to go @gonuke

**Comment by gonuke** [comment]
What's this failing test about??? is that new?

**Comment by bam241** [comment]
it is the same as usual that fail some time.... (I didn't cause that problem :) )

I restarted it

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
LGTM - thanks @bam241

**Pull request merged**